### ISSUE TITLE
enhancement

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#642]
```
What steps will reproduce the problem?
1.open file
2.reading file 
3.want to make a comment or note but no option.

What is the expected output?
Option in a drop down to turn on comment tools

What do you see instead?
Not avalable

What version of the product are you using?
1.6.3

On what operating system?
Android 4.2.2

What device are you using?
Sony Xpirea Tablet Z

Please provide any additional information below.
would pay for this feature with your user interface

```

Original issue reported on code.google.com by `tim...@gmail.com` on 13 Nov 2013 at 4:58

**Comment by GoogleCodeExporter** [comment]
```
this is already a feature
```

Original comment by `geocip...@gmail.com` on 15 Nov 2013 at 11:37

**Comment by GoogleCodeExporter** [comment]
```
Implemented since 2.0
```

Original comment by `Andrei.K...@gmail.com` on 15 Nov 2013 at 12:09
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
```
that is odd because i cant seam to find that functionality. what menu would 
that be under?
```

Original comment by `tim...@gmail.com` on 16 Nov 2013 at 8:32
